app/liquid_tags/unified_embed.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "./unified_embed/tag"
```

Comment:
it's almost like a little baby gem being born here

Reply:
but why does tag (a name we don't use here) need to be loaded to register (which also doesn't seem to reference Tag)?

Reply:
https://github.com/forem/forem/blob/main/config/initializers/liquid.rb#L7-L13 seems to be the thing that's loading for all the other tags - this feels like a good case for moving that `register_tag` statement to the initializer?

Reply:
I introduced this problem as part of #15911.  And in talking with @msarit, I didn't want to have the `Liquid::Template.register_tag("embed", UnifiedEmbed::Tag)` declaration at the bottom of the UnifiedEmbed module.   That breaks with the current pattern (e.g. that which renders the tag registers itself).  So, if we were to do anything, I'd want to consider it for all liquid tags.

My personal architectural preference is to not have the file that declares the class to also register the class.  Instead favor an initializer.  But that is a larger conversation.